modules/virt-editing-vm-yaml-web.adoc

```diff
@@ -1,14 +1,24 @@
 // Module included in the following assemblies:
 //
 // * virt/virtual_machines/virt-edit-vms.adoc
+//Establishing conditional expressions
+//
+ifeval::["{context}" == "virt-edit-vms"]
+:object: virtual machine
+:object-vm-details: Virtual Machine Details
+:object-vm-overview: Virtual Machine Overview
+endif::[]
+
+ifeval::["{context}" == "virt-editing-vm-template"]
+:object: virtual machine template
+:object-vm-details: Virtual Machine Template Details
+endif::[]
 
 [id="virt-editing-vm-yaml-web_{context}"]
 
 = Editing a virtual machine YAML configuration using the web console
 
-Using the web console, edit the YAML configuration of a virtual machine.
-
-Not all parameters can be updated. If you edit values that cannot be changed and click *Save*, an error message indicates the parameter that was not able to be updated.
+Using the web console, edit the YAML configuration of a virtual machine. Some parameters are not editable. If you edit values that cannot be changed and click *Save*, an error message indicates the parameter that was not able to be updated.
```

Comment:
Suggestion: You can edit the YAML configuration of a virtual machine in the web console. Some parameters cannot be modified. If you click *Save* with an invalid configuration, an error message indicates the parameter that cannot be changed.

Reply:
Change made.